copier/main.py

```diff
@@ -25,6 +25,8 @@
 from pydantic.dataclasses import dataclass
 from pydantic.json import pydantic_encoder
 from questionary import unsafe_prompt
+from rich.console import Console
+from rich.markdown import Markdown
```

Comment:
I'm double-thinking about this.

I really like it that we can render markdown in the terminal. However, I'm worried about the lack of consistency.

I can imagine somebody writing an intro message in markdown and a question message also in markdown and wondering... why one works and the other doesn't?

Do you think it would be good to defer markdown support until we can add it consistently?

Other than that, the PR is great.

Reply:
I've looked into supporting Markdown in `questionary` help text through `rich`, but it seems this is currently not possible because `questionary` applies its own styles through `prompt-toolkit` to the message. So when I render it first with `rich` and pass it to `questionary`, then the formatting remains but the styling gets overwritten by `questionary`/`prompt-toolkit`. For instance, currently the help text is printed in bold font, and then also the rendered Markdown becomes bold font. Disabling `questionary`/`prompt-toolkit` styles for help text seems to be impossible at the moment.

I also thought about replacing `rich` by `prompt-toolkit` to avoid introducing yet another printing library, but `prompt-toolkit` doesn't support Markdown rendering. `questionary` is heavily based on `prompt-toolkit`, so converging on a common toolstack would mean we shouldn't use `rich`.

Perhaps we need to answer a few more fundamental questions:

1. How important is Markdown rendering for pre/post copy messages?
2. How important is Markdown rendering for help text?
3. How important is having consistent rendering capabilities? I.e., would we only want Markdown rendering at all when all relevant texts support it?
4. Is it possible to achieve a unified look and feel when using `prompt-toolkit` and `rich` in different places?

We could defer answering those questions by simply removing Markdown support for pre/post copy messages now (and, thus, remove `rich` again), so we can merge this PR more quickly.

WDYT?

Reply:
I've decided to remove `rich` and Markdown support again until we have a proper solution to render Markdown consistently in other places as well.

Reply:
I think it's the best solution for the moment.